src/main/scala/com/comcast/xfinity/sirius/uberstore/segmented/SegmentedUberStore.scala

```diff
@@ -295,7 +304,7 @@ class SegmentedUberStore private[segmented] (base: JFile, eventsPerSegment: Long
     readOnlyDirs = replaceElement(readOnlyDirs, original, newSegment)
   }
 
-  private def removeSegment(segment: Segment) = replaceLock.synchronized {
+  private def replaceSegment(segment: Segment) = replaceLock.synchronized {
```

Comment:
This should probably still be `removeSegment`, right? We're not doing anything other than deleting it.

Reply:
Dang it, IntelliJ...